src/components/Tabs/Tabs.tsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import { Tab } from '../Tab/Tab';
+
+type Props = {
+  selectedTabId: string | undefined
+};
+
+export const Tabs: React.FC<Props> = ({ selectedTabId }) => {
+  const tabs = [
+    { id: 'tab-1', title: 'Tab 1', content: 'Some text 1' },
+    { id: 'tab-2', title: 'Tab 2', content: 'Some text 2' },
+    { id: 'tab-3', title: 'Tab 3', content: 'Some text 3' },
+  ];
+
+  const selectedTab = tabs.find(tab => tab.id === selectedTabId);
```

Comment:
```suggestion
  const tabContent = tabs
    .find(tab => tab.id === selectedTabId)?.content || 'Please select a tab';
```

Just get content here